CHANGELOG.md

```
@@ -2,6 +2,11 @@
 
 This contains all the major changes between versions.
 
+### Unreleased
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)